docs/user-guide/lns-module-configuration.md

```diff
@@ -0,0 +1,38 @@
+| RX2_DATR                    | RX2 data rate                                                                          | No (defaults to 1)                  |
+| RX2_FREQ                    | RX2 frequency                                                                          | No                                  |
+| IOTEDGE_TIMEOUT             | IoT Edge timeout in milliseconds                                                       | No (defaults to 0)                  |
+| FACADE_SERVER_URL           | Azure Facade function url                                                              | Yes                                 |
+| FACADE_AUTH_CODE            | Azure Facade function auth code                                                        | Yes                                 |
+| LOG_LEVEL                   | Logging level                                                                          | No (defaults to level 4 (Error)     |
+| LOG_TO_CONSOLE              | Indicates whether logging to console is enabled or not                                 | No (default to `true`)              |
+| LOG_TO_TCP                  | Indicates whether logging to TCP is enabled or not (used mainly for integration tests) | No (defaults to `false`)            |
+| LOG_TO_HUB                  | Indicates whether logging to IoT Hub is enabled or not                                 | No (defaults to `false`)            |
+| LOG_TO_TCP_ADDRESS          | IP address for TCP logs                                                                | Yes, only if TCP logging is enabled |
+| LOG_TO_TCP_PORT             | TCP port to send logs to                                                               | Yes, only if TCP logging is enabled |
+| NETID                       | Gateway network ID                                                                     | No (defaults to network id 1)       |
+| AllowedDevAddresses         | Allowed Dev addresses                                                                  | No                                  |
+| LNS_SERVER_PFX_PATH         | Path of the .pfx certificate to be used for LNS Server endpoint                        | No                                  |
+| LNS_SERVER_PFX_PASSWORD     | Password of the .pfx certificate to be used for LNS Server endpoint                    | No                                  |
+| CLIENT_CERTIFICATE_MODE     | Specifices the client certificate mode with which the server should be run             | No (defaults to `NoCertificate`)    |
+| LNS_VERSION                 | Version of the LNS                                                                     | No                                  |
+| IOTHUB_CONNECTION_POOL_SIZE | IoT Hub connection pool size                                                           | No (defaults to 1)                  |
```

Comment:
Description: This sets the AMQP Connection Pool Size for communication to IoT Edge Hub / IoT Hub (depending on ENABLE_GATEWAY). Increasing this value to higher number will improve scalability (link to scalability documentation user-guide/scalability)